tests/mongodb.spec.js

```diff
@@ -0,0 +1,48 @@
+const should = require('should');
+const mongoose = require('mongoose');
+const dailyWeathers = require("../models/weatherSchema");
```

Comment:
Also, I think the name of the class should be "Weather", whichis more general then "DailyWeather".